e2e-tests/common/methods.py

```diff
@@ -1,10 +1,16 @@
 import os
+
 import requests
-from common.constants import ANONYMIZER_BASE_URL, ANALYZER_BASE_URL
+
+from common.constants import ANONYMIZER_BASE_URL, ANALYZER_BASE_URL, \
+    IMAGE_REDACTOR_BASE_URL
 
 DEFAULT_HEADERS = {"Content-Type": "application/json"}
+MULTIPART_HEADERS = {"Content-Type": "multipart/form-data"}
 ANALYZER_BASE_URL = os.environ.get("ANALYZER_BASE_URL", ANALYZER_BASE_URL)
 ANONYMIZER_BASE_URL = os.environ.get("ANONYMIZER_BASE_URL", ANONYMIZER_BASE_URL)
+IMAGE_REDACTOR_BASE_URL = os.environ.get("IMAGE_REDACTOR_BASE_URL",
```

Comment:
The other side of reading the url from env var, is setting it in the CI pipeline when smoke testing a deployed instance of our services.
In analyzer and anonymizer cases we run @integration marked tests on the staging slot during deployment, and I think it might be worth to choose one happy scenario that you've added and mark it as integration just to have a smoke test for image redactor. See deploy-stage.yml:103 for reference.

Reply:
I want to make sure since we are not really defining the image redactor as 100% ready yet...
@navalev WDYT should we add smoke test for the image aredactor

Reply:
Currently we do not add the image redactor to the prod demo site and i would avoid it in the integration tests as the tesseract issue is still open. Can be added in future versions once its more stable